src/apps/material/helper.ts

```diff
@@ -48,52 +54,70 @@ export const getWorkDescriptionListData = ({
   const allLanguages = mainLanguages
     .map((language) => language.display)
     .join(", ");
-  const fallBackManifestation = getWorkManifestation(work);
+  const fallBackManifestation = getWorkManifestation(
+    work,
+    "bestRepresentation"
+  );
   const creatorsText = creatorsToString(
     flattenCreators(filterCreators(creators, ["Person"])),
     t
   );
 
   return [
     {
-      label: t("typeText"),
+      label: t("languageText"),
+      value: allLanguages,
+      type: "standard"
+    },
+    {
+      label: t("playTimeText"),
       value:
-        (manifestation?.materialTypes?.[0]?.specific && "") ||
-        (fallBackManifestation?.materialTypes?.[0]?.specific && ""),
+        String(manifestation?.physicalDescriptions?.[0]?.playingTime ?? "") ||
+        String(
+          fallBackManifestation?.physicalDescriptions?.[0]?.playingTime ?? ""
+        ),
       type: "standard"
     },
     {
-      label: t("languageText"),
-      value: allLanguages,
+      label: t("editionText"),
+      value:
+        (manifestation?.edition?.summary ?? "") ||
+        (fallBackManifestation?.edition?.summary ?? ""),
       type: "standard"
     },
+
     {
       label: t("genreAndFormText"),
       value:
         (manifestation?.genreAndForm?.[0] ?? "") ||
         (fallBackManifestation?.genreAndForm?.[0] ?? ""),
       type: "standard"
     },
-    { label: t("contributorsText"), value: creatorsText, type: "link" },
     {
       label: t("originalTitleText"),
-      value: titles && workYear ? `${titles?.original} ${workYear.year}` : "",
+      value: titles?.original?.[0] ?? "",
       type: "standard"
     },
     {
-      label: t("isbnText"),
+      label: t("publisherText"),
       value:
-        (manifestation?.identifiers?.[0]?.value ?? "") ||
-        (fallBackManifestation?.identifiers?.[0]?.value ?? ""),
+        (manifestation?.publisher.join(" / ") ?? "") ||
+        (fallBackManifestation?.publisher.join(" / ") ?? ""),
       type: "standard"
     },
     {
-      label: t("editionText"),
+      label: t("firstEditionYearText"),
+      value: String(workYear?.year ?? t("unknownText")),
```

Comment:
Both text identifiers are pretty generic, especially "unknownText". Could the be prefixd with the context?
Please remember to put them in the proptypes of the app.